MAUI/Maps/zoom-pan.md

```diff
@@ -0,0 +1,633 @@
+---
+layout: post
+title: Zooming and Panning  in Maps control | Syncfusion
+description: Learn here all about the Shape Sublayer feature of the Syncfusion .NET MAUI Maps (SfMaps) control to customize their appearances and more.
+platform: MAUI
+control: SfMaps
+documentation: ug
+---
+
+# Zooming and Panning in .NET MAUI Maps (SfMaps)
+
+It is possible to zoom in and out for any layer to take a closer look at a specific region by pinching the map, scrolling the mouse wheel or track pad. Pan the map to navigate across the regions. You can also customize the zoom level of the rendering.
```

Comment:
It is possible to zoom in and out for any layer to take a closer look at a specific region by pinching the map or scrolling the mouse wheel or trackpad. Pan the map to navigate across the regions. You can also customize the zoom level of the rendering.

Reply:
Changed the content

Reply:
@Shalini-Ashokan `trackpad` is single word.

Reply:
Changed